mkdocs.yaml

```diff
@@ -55,8 +55,7 @@ plugins:
       regex:
         - ^(?!(.*/)?assets/).*\.(?!(.*\.)?md|(.*\.)?svg|(.*\.)?png|(.*\.)?gif|(.*\.)?jpg).*$
         - ^(.*/)?[^.]*$
-  - macros:
-      module_name: mkdocs_macros
```

Comment:
Hmm, I will fix this first: https://github.com/autowarefoundation/autoware.universe/pull/5213

cc. @mitsudome-r

Reply:
Resolved in:
- https://github.com/autowarefoundation/autoware.universe/pull/5475